### PR TITLE
chore(pull_request_template): adds breaking changes section to github template

### PR DESCRIPTION
**Pull request by seaerchin**
## Problem
Devs are supposed to indicate breaking changes for easier rollback if hiccups occur on the release day. This makes it prone to human error, where the devs forget. 

## Solution
adds a breaking change section to the PR template

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ Closes [insert issue #]
 ## Solution
 <!-- How did you solve the problem? -->
 
+**Breaking Changes** 
+<!-- Does this PR contain any backward incompatible changes? If so, what are they and should there be special considerations for release? -->
+- Details ...
```
Comment (karrui) [inline .github/pull_request_template.md:11]: Maybe add a checkbox also for ease of use (like in many open source issue templates)

- [ ] Yes
- [ ] No

Comment (seaerchin) [reply .github/pull_request_template.md:11]: i think add numbers bah so it's clear cos we want to know the breaking change(s) ideally

Comment (karrui) [reply .github/pull_request_template.md:11]: i mean, yes/no is just a bird's eye view, the details part is still there haha

Comment (seaerchin) [reply .github/pull_request_template.md:11]: oh i thought the PR author will delete the section if no breaking changes HAHAHA ok will change!!

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -8,12 +8,12 @@ Closes [insert issue #]
 
 **Breaking Changes** 
 <!-- Does this PR contain any backward incompatible changes? If so, what are they and should there be special considerations for release? -->
+- [ ] Yes - this PR contains breaking changes
+- [ ] No - this PR is backwards compatible  
 - Details ...
 
 **Features**:
 
-- Details ...
```
Comment (karrui) [inline .github/pull_request_template.md:11]: why is this removed?

Comment (seaerchin) [reply .github/pull_request_template.md:11]: my bad, deleted by accident

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -6,6 +6,12 @@ Closes [insert issue #]
 ## Solution
 <!-- How did you solve the problem? -->
 
+**Breaking Changes** 
+<!-- Does this PR contain any backward incompatible changes? If so, what are they and should there be special considerations for release? -->
+- [ ] Yes - this PR contains breaking changes
+- [ ] No - this PR is backwards compatible  
+- Details ...
```
Comment (karrui) [inline .github/pull_request_template.md:13]: nit: extra line break here?

**Review by karrui — APPROVED** [review]
lgtm with niot

File: .github/pull_request_template.md (hunk 3 above)
Comment (seaerchin) [reply .github/pull_request_template.md:13]: oddly enough the formatting broke on preview on github, which is why i omitted it

**without extra line**
Screenshot from 2021-05-18 12-00-26

**with extra line**
Screenshot from 2021-05-18 12-00-55

Comment (karrui) [reply .github/pull_request_template.md:13]: Can move the details under the yes checkbox then

Comment (seaerchin) [reply .github/pull_request_template.md]: fixed!
